notebook_utils/berdl_notebook_utils/minio_governance/operations.py

```diff
@@ -246,6 +248,22 @@ def get_my_policies() -> UserPoliciesResponse:
     return get_my_policies_workspaces_me_policies_get.sync(client=client)
 
 
+def get_my_groups() -> UserGroupsResponse:
+    """
+    Get list of groups the current user belongs to.
+
+    Returns:
+        UserGroupsResponse with username, groups list, and group_count
+
+    Example:
```

Comment:
This example explaining how to iterate over a list is not necessary

Reply:
👍